packages/astro/src/transform/babel.ts

```diff
@@ -0,0 +1,17 @@
+import traverse from '@babel/traverse';
```

Comment:
nit on `src/transform` directory: is this meant to be generally usable across the rest of the codebase? I like the idea of keeping this new code a bit separate/quarantined from the rest of the codebase until it has a chance to settle, given its last-minute nature.

Thoughts on `src/core/add` to match `src/core/dev|preview|build` ?